platform-server-kernel/src/main/java/uno/cod/platform/server/core/config/StorageConfig.java

```diff
@@ -21,10 +21,9 @@
     private StorageProperties storageProperties;
 
     @Bean
-    @Profile(Profiles.APPENGINE)
```

Comment:
@Exordian is this okay? Looks like this way we'd use the same Bean implementation for both AppEngine (in cloud) and local. Didn't you specifically introduce this to override local behaviour?